src/main/java/com/deque/axe/AXE.java

```diff
@@ -61,22 +61,34 @@ private static String getContents(final URL script) {
 	}
 
 	/**
-	 * Recursively injects aXe into all iframes and the top level document.
-	 *
+	 * Recursively injects a script to the top level document with the option to skip iframes.
 	 * @param driver WebDriver instance to inject into
+	 * @param scriptUrl URL to the script to inject
+	 * @param skipFrames True if the script should not be injected into iframes
 	 */
-	public static void inject(final WebDriver driver, final URL scriptUrl) {
+	public static void inject(final WebDriver driver, final URL scriptUrl, Boolean skipFrames) {
 		final String script = getContents(scriptUrl);
-		final ArrayList<WebElement> parents = new ArrayList<WebElement>();
 
-		injectIntoFrames(driver, script, parents);
+		if (!skipFrames) {
+			final ArrayList<WebElement> parents = new ArrayList<WebElement>();
+			injectIntoFrames(driver, script, parents);
+		}
 
 		JavascriptExecutor js = (JavascriptExecutor) driver;
 		driver.switchTo().defaultContent();
 		js.executeScript(script);
 	}
 
 	/**
+	 * Recursively injects a script into all iframes and the top level document.
+	 * @param driver WebDriver instance to inject into
+	 * @param scriptUrl URL to the script to inject
+	 */
+	public static void inject(final WebDriver driver, final URL scriptUrl) {
```

Comment:
Is there a reason why the same function name is defined twice in a row here? I'm surprised that works, although I'm not super experienced with Java.

Reply:
This is 100% normal. JavaScript lets you set default parameter values and leave arguments undefined, but Java requires a method invocation to supply all parameters and expects you to "overload" multiple signatures like this when it's appropriate to call a method with different set of parameters. If the only signature for `inject` had three parameters, a call which supplied two instead (or three parameters of a different type, since Java's strongly typed) wouldn't even compile.

Reply:
Hi, sorry for asking again, but is it possible to move forward with this change? Thanks!

Reply:
@marcysutton @dmfay Please approve if you think this is ready.